src/net/sourceforge/kolmafia/textui/parsetree/Symbol.java

```diff
@@ -37,10 +37,11 @@
 	extends ParseTreeNode
 	implements Comparable<Symbol>
 {
-	public String name;
+	public final String name;
 
 	public Symbol()
```

Comment:
Does anything even use this constructor? Can we get rid of it?

Reply:
Yeah, IDK why it's even there